int32 type
int32 UNSPECIFIED = 0
int32 DISCRETE_JOINT_SPACE = 1
int32 DISCRETE_CARTESIAN_SPACE = 2
int32 DISCRETE_CARTESIAN_AND_JOINT_SPACE = 3
int32 NUM_TYPES = 4